scripts/install_zeromq.sh

```diff
@@ -1,59 +1,46 @@
 #!/bin/bash
 
+cd $SIMPATH/basics
+
 if [ ! -d  $SIMPATH/basics/zeromq ];
 then
-  cd $SIMPATH/basics
-  if [ ! -e zeromq-$ZEROMQVERSION.tar.gz ];
-  then
-    echo "*** Downloading zeromq sources ***"
-    download_file $ZEROMQ_LOCATION/zeromq-$ZEROMQVERSION.tar.gz
-  fi
-  untar zeromq zeromq-$ZEROMQVERSION.tar.gz
-  if [ -d zeromq-$ZEROMQDIR ];
-  then
-    ln -s zeromq-$ZEROMQDIR zeromq
-  fi
+  git clone $ZEROMQ_LOCATION zeromq
 fi
 
 install_prefix=$SIMPATH_INSTALL
 checkfile=$install_prefix/lib/libzmq.a
 
 if (not_there zeromq $checkfile);
 then
-    cd $SIMPATH/basics/zeromq
-
-#    mypatch ../zeromq_clang_c++11.patch
-
-    distribution=$(lsb_release -is)
-    version=$(lsb_release -rs | cut -f1 -d.)     
-
-    if [ "$arch" == "ppc64le" ];then
-     ./autogen.sh
-    fi
-
-#    if [ "$distribution$version" = "ScientificCERNSLC6" ]; then
-      PKG_CONFIG_PATH=$SIMPATH_INSTALL/lib/pkgconfig ./configure --prefix=$install_prefix --libdir=$install_prefix/lib --enable-static --without-libsodium
-#    else
-#      PKG_CONFIG_PATH=$SIMPATH_INSTALL/lib/pkgconfig ./configure --prefix=$install_prefix --libdir=$install_prefix/lib --enable-static
-#    fi
-    
-    make
-    make install
+  cd zeromq
+  git checkout $ZEROMQ_VERSION
+  if [ ! -d  build ];
+  then
+    mkdir build
+  fi
+  cd build
 
-    check_all_libraries  $install_prefix/lib
+  cmake .. \
+        -DWITH_PERF_TOOL=OFF \
```

Comment:
Let's turn this on, the perf tools are useful sometimes to compare performance and the build times are not very high.

Reply:
@rbx Done